tests/casp/journal_check.pm

```diff
@@ -0,0 +1,73 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Basic CASP journal tests
+# Maintainer: Tomas Hehejik <thehejik@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub write_detail_output {
+    my ($self, $title, $output, $result) = @_;
+
+    my $filename = $self->next_resultname('txt');
+    my $detail   = {
+        title  => $title,
+        result => $result,
+        text   => $filename,
+    };
+    push @{$self->{details}}, {result => 'unk'};    # Is it needed?
+    push @{$self->{details}}, $detail;
+
+    open my $fh, '>', bmwqemu::result_dir() . "/$filename";
+    print $fh $output;
+    close $fh;
+
+    # Set overall result for the test, values are "ok" and "fail", result in details table has value "softfail" in addition
+    # when result in details is "ok" then overall result is "undef" and still the result in webUI is calculated as "passed"
+    $self->{result} = "fail" if $result eq "fail";
```

Comment:
Still not sure about this (mark overall result as failed but only if one of the "detail" is "fail" - seems it works correctly for important=1 tests).